app/assets/stylesheets/components/_cards.scss

```diff
@@ -8,13 +8,22 @@
   background-position: center;
   background-size: cover;
   min-width: 0;
-  width: 200px;
-  height: 100px;
+
+  @media screen and (min-width: $small) {
+    width: 250px;
+    height: 125px;
+    font-size: 20px;
+  }
+
+  @media screen and (min-width: $large) {
+    width: 200px;
+    height: 100px;
+    font-size: 16px;
+  }
 }
```

Comment:
You need a base width and height otherwise from 0px to 560px screens there's nothing

Also why is the size decreasing as the screen's size grow ?